Makefile

```diff
@@ -102,6 +102,12 @@ test-ui-debug: lint-ui
 
 test: test-server test-ui ## Run all tests
 
+docs-dependencies:
+	@pip install -q -r docs/requirements.txt
+
+docs: docs-dependencies
```

Comment:
Let's add the "double-hashed" comment to indicate this target is used to build documentation.  e.g., `## Build docs`.
Such comments then show when typing `make` as a form of help text.

Reply:
Fixed.